.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,32 @@
+<!--
+Before opening a new issue, please ensure:
+- You use the forums for support/questions
+- You search for existing bugs/feature requests
+-->
 
+## Support / Questions
 
+Please use https://forums.sonarr.tv/ for support. Support requests or questions will be redirected to the forums and the issue will be closed.
+
+## Bug Report
+**Sonarr Version:**
+**Operating System:**
+**.net Framework (Windows) or mono (macOS/Linux) Version:**
+**Link to Log Files (debug or trace):**
+**Browser (for UI bugs):**
+
+### Steps to reproduce
+1.
+
+### Current Behavior
+
+
+### Expected Behavior
+
+
+## Feature Request
+
+###What problem are you looking to solve
```

Comment:
I think there needs to be a space here (between the `###What` for proper Github markdown support. It's certainly necessary for visual clarity :)